### PR TITLE
Add iOS recruiting project

### PR DESCRIPTION
**Pull request by stephenfwalsh**

### CODE REVIEW

File: Flipdish.Recruiting.iOS/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+We are happy you're interested in an engineering position at Flipdish!
+
+### About This Challenge
+We want to see how you construct software, but we also know that you're busy. So, rather than ask you to build something from scratch, we want you to take whatever time is reasonable and refactor some of the code in this application. Just make it better, in literally any way that you see fit.
+
+It doesn't matter how long you spend on improving it! You could spend 5 minutes, or 5 hours -- the amount of time is totally up to you. It'll really help us understand your perspectives on software when we see how you improve existing code that you haven't encountered before. Feel free to refactor, change, improve, rewrite or optimize any part of the application at all.
+
+### About The app
+
+The app is a very simple List / Detail client for the [The Dog API](https://docs.thedogapi.com/). It works (for the most part), but the code is truly, madly and deeply squalid. As you'll quickly notice, the app is far from perfect in many different ways - Let's make it better!
```
Comment (brikelly) [inline Flipdish.Recruiting.iOS/README.md]: Duplicate of "the" in here in the link to the API?

File: Flipdish.Recruiting.iOS/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+##### Functionality
+
+- The basic premise is to fetch a list of Dogs from the API, and render them in a list.
+- Each list item can be opened for a detail view and from there the user can favourite their favourite Perros 🦮🐕‍🦺🐕🐩.
+- There's a section at the top of the list view with quick access to the user's favourited Dogs.
+
+See attached screenshots for how the app <em>should</em> look.
```
Comment (brikelly) [inline Flipdish.Recruiting.iOS/README.md:18]: Does this `<em>` work in GitHub markdown? That's where people will read this file.

File: Flipdish.Recruiting.iOS/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+It doesn't matter how long you spend on improving it! You could spend 5 minutes, or 5 hours -- the amount of time is totally up to you. It'll really help us understand your perspectives on software when we see how you improve existing code that you haven't encountered before. Feel free to refactor, change, improve, rewrite or optimize any part of the application at all.
+
+### About The app
+
+The app is a very simple List / Detail client for the [The Dog API](https://docs.thedogapi.com/). It works (for the most part), but the code is truly, madly and deeply squalid. As you'll quickly notice, the app is far from perfect in many different ways - Let's make it better!
+
+##### Functionality
+
+- The basic premise is to fetch a list of Dogs from the API, and render them in a list.
+- Each list item can be opened for a detail view and from there the user can favourite their favourite Perros 🦮🐕‍🦺🐕🐩.
```
Comment (brikelly) [inline Flipdish.Recruiting.iOS/README.md]: Duplicate of `favourite` reads weird, maybe say "choose their favourite..."?

File: Flipdish.Recruiting.iOS/README.md (hunk 1 above)
Comment (brikelly) [reply Flipdish.Recruiting.iOS/README.md]: Also, maybe don't scare people with the "truly, madly, squalid" language (although I love the phrasing!). Perhaps something like "but the code has a lot of issues".

**Review by brikelly — CHANGES_REQUESTED** [review]

File: Flipdish.Recruiting.iOS/README.md (hunk 2 above)
Comment (stephenfwalsh) [reply Flipdish.Recruiting.iOS/README.md:18]: It does surprisingly! For italicising, if you open this page you should see it working https://github.com/flipdishbytes/recruiting/tree/add-ios-recruiting-project/Flipdish.Recruiting.iOS

File: Flipdish.Recruiting.iOS/README.md (hunk 3 above)
Comment (stephenfwalsh) [reply Flipdish.Recruiting.iOS/README.md]: Fixed 😄

File: Flipdish.Recruiting.iOS/README.md (hunk 1 above)
Comment (stephenfwalsh) [reply Flipdish.Recruiting.iOS/README.md]: Hahaha yes, the language was a bit too dramatic there 😆 , fixed now 😄

**Comment by stephenfwalsh** [comment]
@brikelly updated `README.md`, see changes above

**Review by brikelly — APPROVED** [review]
LGTM!

**Pull request merged**